Dockerfile

```diff
@@ -59,3 +61,5 @@ RUN apt-get -y install debootstrap
 
 # Install shell check
 RUN apt-get -y --no-install-recommends install shellcheck
+
+RUN echo "{\"rev\":\"$GIT_COMMIT\",\"branch\":\"${GIT_BRANCH}\"}" > /buildinfo.json
```

Comment:
How is buildinfo supposed to be used? To we want to commit it to the repo?

Reply:
This is my proposal for automatically track at which commit this docker was built. At build time, this file is created with the relevant information, and if you want to know at which commit this image was built you just need to check this file. No manual actions needed, no need to store anything in the repo itself. WDYT?